src/components/ExpansionPanel/ExpansionPanel.test.tsx

```diff
@@ -11,7 +12,7 @@ describe('ExpansionPanel', () => {
       </ExpansionPanel>
     );
 
-    expect(wrapper.prop('expanded')).toBe(false);
+    expect(wrapper.find(Collapse).prop('isOpen')).toBe(false);
```

Comment:
here we are testing implementation details, wouldn't it be better if we assert the content is not visible?

Reply:
Probably. I was just trying to get tests to pass rather than trying to pass for this change rather than testing things better. We will have React `testing-library` added soon with some of @pixelbandito's changes, so that will be a set in the right direction for that.

Also, the content is still there in the HTML, but the CSS changes, so we would still be asserting against CSS selectors.